rules/imports.js

```diff
@@ -18,13 +18,13 @@ module.exports = {
   settings: {
     'import/resolver': {
       node: {
-        extensions: ['.js', '.mjs', '.json', 'ts', 'mts'],
+        extensions: ['.js', '.mjs', '.json', '.ts', '.mts'],
```

Comment:
Is it not necessary to add '.tsx' here?
If not, please state the reason in the code comments or in the PR description.

Reply:
Considering RSC it would be appropriate to add them as well, so I'd added them.